src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -280,6 +294,25 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
           return;
         }
 
+      case PAGE_UP:
+        if (this._pageUpAndDown.enabled && isModifierAllowed) {
+          const targetIndex = this._activeItemIndex - this._pageUpAndDown.delta;
+          this._setActiveItemByIndex(targetIndex > 0 ? targetIndex : 0, 1);
```

Comment:
You should be able to simplify this and the PAGE_DOWN handling using the `_setActiveItemByDelta` method instead.

Reply:
During writing the code I first had the same intention, but in my opinion this method can only be used to move 1 up or down and not an defined number or did i misunderstand anything?

Reply:
I see. Let's keep it like this for now.